hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineProvider.java

```diff
@@ -18,12 +18,6 @@
 
 package org.apache.hadoop.hdds.scm.pipeline;
 
-import java.io.IOException;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
```

Comment:
Please avoid unnecessarily reordering imports.  It creates noise in the patch and, if merged, can cause conflicts for other changes (current PRs or when backporting).